lib/lte_link_control/lte_lc.c

```diff
@@ -814,13 +814,22 @@ int lte_lc_normal(void)
 
 int lte_lc_psm_param_set(const char *rptau, const char *rat)
 {
-	if (rptau == NULL || strlen(rptau) != 8 ||
-		rat == NULL || strlen(rat) != 8) {
-		return -EINVAL;
+	if (rptau != NULL && strlen(rptau) == 8) {
+		memcpy(psm_param_rptau, rptau, sizeof(psm_param_rptau));
+		LOG_DBG("RPTAU set to %s", log_strdup(psm_param_rptau));
+		if (rat != NULL && strlen(rat) == 8) {
+			memcpy(psm_param_rat, rat, sizeof(psm_param_rat));
+			LOG_DBG("RAT set to %s", log_strdup(psm_param_rat));
+		} else {
+			memset(psm_param_rat, 0x00, sizeof(psm_param_rat));
+			LOG_DBG("RAT use default");
+		}
+	} else {
+		memset(psm_param_rptau, 0x00, sizeof(psm_param_rptau));
```

Comment:
Since we're checking the string length of these buffers, it should be sufficient to set the first byte to zero to save a few cycles, but not a big deal